vertical-pod-autoscaler/e2e/v1/recommender.go

```diff
@@ -254,6 +254,64 @@ var _ = RecommenderE2eDescribe("VPA CRD object", func() {
 	})
 })
 
+var _ = RecommenderE2eDescribe("VPA CRD object", func() {
+	f := framework.NewDefaultFramework("vertical-pod-autoscaling")
+
+	var (
+		vpaClientSet vpa_clientset.Interface
+	)
+
+	ginkgo.BeforeEach(func() {
+		vpaClientSet = getVpaClientSet(f)
+	})
+
+	ginkgo.It("with all container recommendations", func() {
```

Comment:
It does what with all container recommendations?

"with no containers opted out all container get recommendations"?